### DISCUSSION TITLE
Not seeing ads when IDFA "ask not to track"

### DISCUSSION THREAD

**Discussion by Xioor** [Poing-Studios/Godot-AdMob-Android-iOS#65] [General]
Hello, I am not seeing ads when I select "ask not to track" in the IDFA prompt. I added over 30 providers to SKAdNetworkItems and I have GADDelayAppMeasurementInit set to true. Any ideas?

**Comment by gumaciel** [comment]
You don't need to add the others `SKAdNetwork`, these: https://developers.google.com/admob/ios/3p-skadnetworks, you just need this: `cstr6suwn9.skadnetwork`

- I am not seeing ads when I select "ask not to track" in the IDFA prompt.
Are you testing with Test Ads and Test Ids? If yes, i don't know why is not working i will check this soon

**Reply by Xioor** [reply]
I switched to my own GADApplicationIdentifier and changed to my own unit_ids. What do you mean by test ads and test ids?

**Reply by gumaciel** [reply]
Try to Change to this GADApplicationIdentifier to the Test: `ca-app-pub-3940256099942544~1458002511` and use the Demo ad units: https://developers.google.com/admob/ios/test-ads#demo_ad_units 

And also: Unmark 'Is Real' on `MobileAds.tscn`

**Reply by Xioor** [reply]
That worked, but it does not work if I put my GADApplicationIdentifier and my own unit_ids 🤔

**Reply by gumaciel** [reply]
Apple has a problem when you are testing with your Ads, your app is released on App Store and linked on AdMob website? This need to be done to your Ads appear

**Reply by gumaciel** [reply]
https://github.com/Poing-Studios/Godot-AdMob-Android-iOS/wiki/Question-and-Answers#4-why-my-real-ads-are-now-showing

Read more here

**Reply by Xioor** [reply]
Oh I see, my game is on the itunes dashboard, but never released yet. Thank you!
